manifests/v1beta1/installs/katib-standalone/kustomization.yaml

```diff
@@ -11,8 +11,10 @@ resources:
   - ../../components/crd/
   # Katib DB manager.
   - ../../components/db-manager/
-  # Katib DB mysql.
-  - ../../components/mysql/
+  # # Katib DB mysql.
+  # - ../../components/mysql/
+  # Katib DB postgres.
+  - ../../components/postgres/
```

Comment:
Should we use Postgres, or MySQL by default?

Reply:
If there is no special reason, just keep MySQL by default is better for now.
Since kubeflow pipeline also use MySQL by default, and we should notify to katib users in advance to prepare the default db will be changed

Reply:
For now, I'll rollback it to only install mysql